app/javascript/dashboard/routes/dashboard/settings/reports/components/ReportFilters.vue

```diff
@@ -0,0 +1,209 @@
+<template>
+  <div class="flex-container flex-dir-column medium-flex-dir-row">
+    <div v-if="type === 'agent'" class="small-12 medium-3 pull-right">
+      <multiselect
+        v-model="currentSelectedFilter"
+        :placeholder="$t('AGENT_REPORTS.FILTER_DROPDOWN_LABEL')"
+        label="name"
+        track-by="id"
+        :options="filterItemsList"
+        :option-height="24"
+        :show-labels="false"
+        @input="changeFilterSelection"
+      >
+        <template slot="singleLabel" slot-scope="props">
+          <img
+            class="reports-option__rounded--item reports-option__item"
+            :src="props.option.thumbnail"
+          />
```

Comment:
Can we use the `Thumbnail` component here? https://github.com/chatwoot/chatwoot/blob/develop/app/javascript/dashboard/components/widgets/Thumbnail.vue

Reply:
Fixed in https://github.com/chatwoot/chatwoot/pull/3084/commits/a80b6189e631d673b74c2dd13899346a8d4de25b